README.md

```diff
@@ -11,6 +11,8 @@ Publishing messages with Q_IT
 
 ```
 
+* _Queue name_ - Should be between 3-80 characters long with any alphanumeric characters and hyphen(-) and underscore(_) are allowed.
```

Comment:
I think `queue_names` are also case sensitive. Please verify and mention it out here.